google-cloud-spanner/acceptance/spanner/client/types/pgjsonb_test.rb

```diff
@@ -0,0 +1,172 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "spanner_helper"
+
+describe "Spanner Client", :types, :json, :spanner do
+  let(:db) { spanner_pg_client }
+  let(:table_name) { "stuffs" }
+  let(:json_params) { { "venue" => "abc", "rating" => 10 } }
+  let :json_array_params do
+    3.times.map do |i|
+      { "venue" => "abc-#{i}", "rating" => 10 + i }
+    end
+  end
+
+  it "writes and reads json" do
+    skip if emulator_enabled?
+
+    id = SecureRandom.int64
```

Comment:
Totally optional, feel free to ignore:

This seems to be consistent across all tests. You could avoid repeating it with:

```ruby
  let(:id) { SecureRandom.int64 }
```

Reply:
I would prefer leaving this as this creates random id for each tests to makes sure that they are isolated and not the same row is used for every test.

Reply:
Just realised the let block is executed each time we access it not just once 👍 Updated the tests.